### PR TITLE
Replaced 4 Codecademy Pro links with a free online resource

### LINKED ISSUE #1115: Replace 3 Codecademy Pro links with something free
I audited them, and most are fine.  Here is a list of the links that need replacing:
- [ ] https://www.codecademy.com/courses/asynchronous-javascript/lessons/promises/exercises/understanding-promises in curriculum/javascript/javascript-9.md
- [ ] https://www.codecademy.com/learn/learn-the-command-line in curriculum/command-line/command-line-interface.md
- [ ] https://www.codecademy.com/learn/learn-jquery in curriculum/jquery/jquery.md
- [ ] https://www.codecademy.com/learn/learn-sass in sass/sass.md (low priority)

@alodahl

### PR DESCRIPTION
**Pull request by abhinashgiri**
fixes #1115
replaced 4 links with a free online resource.

### CODE REVIEW

File: javascript/javascript-9-async.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ askMom();
 
 ### Supplemental Materials
 - [Using Promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_promises)
-- [Understanding Promises](https://www.codecademy.com/courses/asynchronous-javascript/lessons/promises/exercises/understanding-promises)
+- [Understanding Promises](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Promises)
```
Comment (Kundan28) [inline javascript/javascript-9-async.md]: 👍

File: sass/sass.md [hunk 2]
```diff
@@ -71,7 +71,7 @@ Sass has been defined as a layer between the stylesheets you create and the .css
 #### Activity 1 - How to set up SaaS
 - Follow steps to [Install Sass](http://sass-lang.com/install)
 #### Activity 2 - Roll your sleeves up and learn Sass with Code Academy
-- Hands on practice with Interactive Lessons [Code Academy](https://www.codecademy.com/learn/learn-sass)
+- Hands on practice [w3schools](https://www.w3schools.com/sass/default.asp)
```
Comment (Kundan28) [inline sass/sass.md]:
```suggestion
- Hands on practice at [w3schools](https://www.w3schools.com/sass/default.asp)
```

File: command-line/command-line-interface.md [hunk 3]
```diff
@@ -188,7 +188,7 @@ Scroll through the manual and see if you can get an idea of what the format of a
 ### Supplemental Materials
 
 - [An introduction to bash](https://guide.bash.academy/inception/). This is a complete guide to bash. It explains its installation and basic concepts, and has been recommended by seasoned developers. 
-- [Codecademy's CLI course](https://www.codecademy.com/learn/learn-the-command-line)
+- [Linux CLI course](http://linuxcommand.org/index.php)
```
Comment (Kundan28) [inline command-line/command-line-interface.md]:
```suggestion
- [Linux CLI course](http://linuxcommand.org/)
```

File: web/jquery.md [hunk 4]
```diff
@@ -92,7 +92,7 @@ For each of the following links, read the code and play around with its function
 ### Supplemental Materials
 - [jQuery official Documentation](https://api.jquery.com/)
 - [Code School's jQuery course](https://www.codeschool.com/courses/try-jquery)
-- [jQuery tutorial on codecademy](https://www.codecademy.com/learn/learn-jquery).
+- [jQuery hands on experience on w3schools](https://www.w3schools.com/jquery/).
```
Comment (Kundan28) [inline web/jquery.md]: 👍

**Comment by abhinashgiri** [comment]
done any more changes required ?

**Comment by Kundan28** [comment]
Updated in scoresheet too.

**Pull request merged**